elide-graphql/src/test/java/com/yahoo/elide/graphql/GraphQLEndpointTest.java

```diff
@@ -433,10 +430,10 @@ void testNonShareable() throws IOException, JSONException {
                                                                 argument("op", enumValue("UPSERT")),
                                                                 argument(
                                                                         "data",
-                                                                        objectValueWithVariable("id: \"123\", " +
+                                                                        enumValue("{id: \"123\", " +
                                                                                 "name: \"my new author\", \" +\n" +
                                                                                 "                " +
-                                                                                "\"noShare:{id:\"1\"}")
+                                                                                "\"noShare:{id:\"1\"}}")
```

Comment:
I'm still thinking about how to inject alien field into Jackson serialization. Maybe I'll put it in the next PR

Reply:
update: my bad. It's not a problem